scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/ExecuterInformationData.java

```diff
@@ -88,22 +88,23 @@ public ExecuterInformation toExecuterInformation(boolean loadFullState) {
                 if (loadFullState) {
                     logger.warn("Task launcher " + taskLauncherNodeUrl + " of task " + taskId +
                                 " cannot be looked up, try to rebind it");
-                    taskLauncher = getReboundTaskLauncher(e);
+                    taskLauncher = getReboundTaskLauncherIfStillExist(e);
                 }
             }
         }
         return new ExecuterInformation(taskLauncher, nodes, nodeName, hostName);
     }
 
-    private TaskLauncher getReboundTaskLauncher(Exception e) {
+    private TaskLauncher getReboundTaskLauncherIfStillExist(Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "e". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)